sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -222,6 +224,78 @@ def recognize_entities(  # type: ignore
         except HttpResponseError as error:
             process_batch_error(error)
 
+    @distributed_trace
+    def recognize_pii_entities(  # type: ignore
+        self,
+        documents,  # type: Union[List[str], List[TextDocumentInput], List[Dict[str, str]]]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Union[RecognizePiiEntitiesResult, DocumentError]]
+        """Recognize entities containing personal information for a batch of documents.
+
+        Returns a list of personal information entities ("SSN",
+        "Bank Account", etc) in the document.  For the list of supported entity types,
+        check https://aka.ms/tanerpii
+        for document length limits, maximum batch size, and supported text encoding.
```

Comment:
```suggestion
```

this sentence got copied twice, it's found below

Reply:
muchas gracias